sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/page_object/android/ApiDemosListViewScreenChaided.java

```diff
@@ -33,48 +29,48 @@
 	 * It was done so just for obviousness
 	 */
 
-	@AndroidFindBys({
+	@AndroidFindBySet({
 		@AndroidFindBy(uiAutomator = "new UiSelector().resourceId(\"android:id/list\")"), //the searching

```

Comment:
Can this be simply multiple repetitive @AndroidFindBy instead of using Set please? That would be more simple example for users.